pyartcd/pyartcd/pipelines/ocp4_scan.py

```diff
@@ -170,16 +170,18 @@ async def _rhcos_inconsistent(self):
 
 @cli.command('ocp4-scan')
 @click.option('--version', required=True, help='OCP version to scan')
+@click.option('--data-path', required=False, default=constants.OCP_BUILD_DATA_URL,
```

Comment:
when do branch cut I used this job to check all images have new branch setup, so I think it's different with released usecase

Reply:
We probably have other ways to check for the existence of the desired target branches, this doesn't sound like a good reason to let scan-sources run against forks (that would trigger ocp4, rhcos, build-sync, etc), and it would be useful only at branch time, dangerous otherwise

Reply:
ok, I will hold this change